pkg/render/apiserver.go

```diff
@@ -1038,3 +1054,67 @@ func (c *apiServerComponent) tigeraNetworkAdminClusterRole() *rbacv1.ClusterRole
 		Rules: rules,
 	}
 }
+
+func (c *apiServerComponent) apiServerPodSecurityPolicy() *policyv1beta1.PodSecurityPolicy {
+	trueBool := true
+	ptrBoolTrue := &trueBool
+	return &policyv1beta1.PodSecurityPolicy{
+		TypeMeta: metav1.TypeMeta{Kind: "PodSecurityPolicy", APIVersion: "policy/v1beta1"},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "tigera-apiserver",
+			Annotations: map[string]string{
+				"seccomp.security.alpha.kubernetes.io/allowedProfileNames": "*",
+			},
+		},
+		Spec: policyv1beta1.PodSecurityPolicySpec{
+			Privileged:               true,
+			AllowPrivilegeEscalation: ptrBoolTrue,
+			RequiredDropCapabilities: []corev1.Capability{
+				corev1.Capability("ALL"),
+			},
+			Volumes: []policyv1beta1.FSType{
+				policyv1beta1.ConfigMap,
+				policyv1beta1.EmptyDir,
+				policyv1beta1.Projected,
+				policyv1beta1.Secret,
+				policyv1beta1.DownwardAPI,
+				policyv1beta1.PersistentVolumeClaim,
+				policyv1beta1.HostPath,
+			},
+			HostNetwork: false,
+			HostPorts: []policyv1beta1.HostPortRange{
+				policyv1beta1.HostPortRange{
+					Min: int32(0),
```

Comment:
The intention was to allow all ports. I believe port 0 is a fake port that could be specified when the system is supposed to assign a free port number. Might be OK to skip since I don't think any components specifically call on it, but since the intention of the policy was to not limit any ports (those should be restricted via network policy), I left all the host ports open.